modules/storage-persistent-storage-efs-pvc.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// storage/persistent-storage/persistent-storage-efs.adoc
+
+[id="efs-pvc_{context}"]
+= Creating the persistent volume claim
+
+.Prerequisites
+
+* The EFS provisioner pod has been created successfully.
+
+.Procedure
+
+. In the {product-title} console, click *Storage* -> *Persistent Volume Claims*.
+
+. In the persistent volume claims overview, click *Create Persistent Volume
+Claim*.
+
+. Define the desired options on the page that appears.
+
+.. Select the storage class created previously from the drop-down menu.
+
+.. Enter a unique name for the storage claim.
+
+.. Select the access mode. This determines the read and write access for the
+created storage claim.
+
+.. Define the size of the storage claim.
++
+[NOTE]
+====
+The size is a requirement; however, every Pod accessing the EFS volume
```

Comment:
What? Seriously? This is a very strange setting.

I'd say something like, "Although you must enter a size, every Pod that access the EFS volume has unlimited storage. Define a value that will remind you that the storage size is unlimited."